nengo_viz/components/netgraph.py

```diff
@@ -42,12 +153,16 @@ def get_parents(self, uid):
         return parents
 
     def update_client(self, client):
+        self.check_for_reload()
```

Comment:
That's a good point.  In ```nengo_gui``` this behaviour got turned on by a command-line option (and wasn't on by default).  Maybe we should do the same here?